cv_next/server/api/supabaseHelper.ts

```diff
@@ -0,0 +1,71 @@
+import "server-only"
+
+import { createClient, SupabaseClient } from "@supabase/supabase-js"
+import {
+  createBrowserClient,
```

Comment:
Do we have es-lint rule for unused imports?

Reply:
Will check

Reply:
Added precommit for unused imports and vars